HISTORY.md

```diff
@@ -26,6 +26,7 @@
 * Fixed a memory safety bug in experimental HyperClockCache (#10768)
 * Fixed some cases where `ldb update_manifest` and `ldb unsafe_remove_sst_file` are not usable because they were requiring the DB files to match the existing manifest state (before updating the manifest to match a desired state).
 * Fix FIFO compaction causing corruption of overlapping seqnos in L0 files due to ingesting files of overlapping seqnos with memtable's under `CompactionOptionsFIFO::allow_compaction=true` or `CompactionOptionsFIFO::age_for_warm>0` or `CompactRange()/CompactFiles()` is used. Before the fix, `force_consistency_checks=true` may catch the corruption before it's exposed to readers, in which case writes returning `Status::Corruption` would be expected.
+* Fix Universal compaction causing corruption of overlapping seqnos in L0 files due to ingesting files of overlapping seqnos with memtable's, including the case when `CompactRange()/CompactFiles()` is used. Before the fix, `force_consistency_checks=true` may catch the corruption before it's exposed to readers, in which case writes returning `Status::Corruption` would be expected.
```

Comment:
@ajkr for the case where CompactRange (and CompactFile ?) can compact to non-L0 then move to L0, is there a specific option to do that? I want to note it in HISTORY that this case is current NOT fixed.

Reply:
Thanks - will specify this situation not being fixed in HISTORY for now.

Reply:
And it seems weird to me that the ReFitLevel https://github.com/facebook/rocksdb/blob/main/db/db_impl/db_impl_compaction_flush.cc#L1204 happens after we compact range with the output level computed **before** but not **after** considering change_level. So like the output_level we use in compact range is not final final.

Reply:
Updated the HISTORY.md to mention this.